java-1.js

```diff
@@ -20,3 +20,148 @@ mytra2.onclick = function () {
+    linktoliveversion: '',
+    linktosource: '#',
+  },
+  {
+    name: 'Tonic',
+    description: `  A daily selection of privately personalized reads; no accounts or
+    sign-ups required.`,
+    featuredImage: './picture/pc3.png',
+    technologies: ['html ', 'css ', 'java script'],
+    linktoliveversion: '',
+    linktosource: '#',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description: `  A daily selection of privately personalized reads; no accounts or
+    sign-ups required.`,
+    featuredImage: './picture/pc4.png',
+    technologies: ['html ', 'css ', 'java script'],
+    linktoliveversion: '',
+    linktosource: '#',
+  },
+];
+
+const project = document.querySelector('#portid');
+
+for (let i = 0; i < projectArray.length; i += 1) {
+  project.innerHTML += `
+  <div class="${i % 2 ? 'repeated-div' : 'repeated-div-2'}">
+  
+
+  
+  <img class="repeated-img" src="${
+  projectArray[i].featuredImage
+}" alt="Snapshoot Portfolio " />
+<img class="pcrepeated-img" src="${
+  projectArray[i].featuredImage
+}" alt="Snapshoot Portfolio">
+<div class="pc12-div">
+  <h3 class="repeated-p1"> ${projectArray[i].name} </h3>
+  <ul class="repeated-ul">
+    <li class="repeated-li1">CANOPY</li>
+    <li class="repeated-li2">Back End Dev</li>
+    <li class="repeated-li3">2015</li>
+  </ul>
+  <p class="repeated-p2">
+  ${projectArray[i].description}
+  </p>
+  <ul class="repeated-ul2">
+    <li class="repeated-li4">${projectArray[i].technologies[0]}</li>
+    <li class="repeated-li5">${projectArray[i].technologies[1]}</li>
+    <li class="repeated-li6">${projectArray[i].technologies[2]}</li>
+  </ul>
+<button class="see-button" type="button">See project</button>
+</div> 
+</div>
+`;
+}
+
+const seebutton = document.querySelectorAll('.see-button');
+// const pc12div = document.querySelectorAll(".pc12-div");
+
+seebutton.forEach((button) => {
+  button.addEventListener('click', () => {
+    const overlay = document.createElement('div');
+    overlay.className = 'popup-overly';
+    document.body.appendChild(overlay);
+
+    const popupbox = document.createElement('div');
+    popupbox.className = 'popup-box';
+    overlay.appendChild(popupbox);
+    for (let i = 0; i < projectArray.length; i += 1) {
+      popupbox.innerHTML += `
+      <div class="popupcardfirst"> 
+      <div class="axey">x</div>
+            <h3 class="repeated-p1"> ${projectArray[i].name} </h3>
+            <div> 
+            <ul class="repeated-ul">
+            <li class="repeated-li1">CANOPY</li>
+            <li class="repeated-li2">Back End Dev</li>
+            <li class="repeated-li3">2015</li>
+            </div> 
+      <div class="${i % 2 ? 'repeated-div' : 'repeated-div-2'}">
+      <img class="repeated-img" src="${
+  projectArray[i].featuredImage
+}" alt="Snapshoot Portfolio " />
+    <img class="pcrepeated-img" src="${
+  projectArray[i].featuredImage
+}" alt="Snapshoot Portfolio"> 
+    </div>
+    <div class="pc12-div">
+    <div>
```

Comment:
- [ ] Each work card has the corresponding popup window with a different title and image, not every popup window is the same for each card 👀. Modal popup data is populated with the data from the project array 👍🏽.